src/topics.js

```diff
@@ -327,6 +329,14 @@ var async = require('async'),
 		});
 	}
 
+	Topics.getUserBookmark = function (tid, uid, callback) {
+		Topics.getTopicField(tid + ':bookmarks', uid, callback);
+	}
+
+	Topics.setUserBookmark = function(data, callback) {
+		Topics.setTopicField(data.tid + ':bookmarks', data.uid, data.postIndex, callback);
+	}
+
```

Comment:
While a clever use of `get`/`setTopicField`, I'd still recommend you use `get`/`setObjectField` for clarity's sake :laughing: